### PR TITLE
skip printing out if library mode

### PR DESCRIPTION
**Pull request by itaysk**
fix: #819

### CODE REVIEW

File: tracee-ebpf/tracee/tracee.go [hunk 1]
```diff
@@ -904,8 +906,9 @@ func (t *Tracee) Run() error {
 	<-sig
 	t.eventsPerfMap.Stop()
 	t.fileWrPerfMap.Stop()
-	t.printer.Epilogue(t.stats)
-
+	if t.config.ChanEvents != nil {
+		t.printer.Epilogue(t.stats)
```
Comment (yanivagman) [inline tracee-ebpf/tracee/tracee.go:910]: I think there is some important information in the epilog that a consumer of tracee as a library may want to know. This information includes, for example, the number of lost events and the event count.

**Comment by grantseltzer** [comment]
What do you think about instead exposing an API where the library consumer can pass a logger interface?

File: tracee-ebpf/tracee/tracee.go (hunk 1 above)
Comment (itaysk) [reply tracee-ebpf/tracee/tracee.go:910]: Sure but this shouldn't be "printed" for the consumer right? 
We should add another API to retrieve that other information. And also I think we should take all the printing out of Tracee library. 
Anyway, this is just a bug fix which should have been included with the original commit

**Comment by itaysk** [comment]
IMO the library should never print stuff, it should only return data and "main" should print it. We can pass a logger for errors though I don't think that for outputs

**Review by yanivagman — APPROVED** [review]

**Pull request merged**